src/algorithm/convexhull.rs

```diff
@@ -0,0 +1,174 @@
+use std::cmp::Ordering;
+use num_traits::Float;
+use types::{Point, Polygon, LineString};
+use std::collections::BTreeSet;
+
+impl<T> Eq for Point<T> where T: Float {}
```

Comment:
Is there some reason you need `Ord`? IIRC, `num::Float` intentionally doesn't implement `Ord` since it can't guarantee total order...I think?